[CI] MacOS builds sometime fail when using glbinding

**SuperTux version:** master or any fork
**System information:** CI (macOS-11, Debug/Release, ON)

##### Expected behavior
Compiles without problems.

##### Actual behavior
```
...
In file included from /usr/local/include/glbinding/gl/gl.h:8:
/usr/local/include/glbinding/gl/types.h:10:10: fatal error: 'KHR/khrplatform.h' file not found
#include <KHR/khrplatform.h>
```
##### Steps to reproduce actual behavior
1. Start workflows.
2. Wait for macOS CIs to start

--- a/.github/workflows/macos.yml
+++ b/.github/workflows/macos.yml
@@ -53,6 +53,7 @@ jobs:
           brew install cmake googletest bash rename libogg libvorbis glew \
                        openal-soft sdl2 sdl2_image sdl2_ttf freetype harfbuzz \
                        fribidi glib gtk-doc glbinding@2 libraqm glm zlib
+          brew link glbinding@2
           install_name_tool -change \
             '@rpath/libsharpyuv.0.dylib' \
             /usr/local/opt/webp/lib/libsharpyuv.0.dylib \

--- a/.github/workflows/macos.yml
+++ b/.github/workflows/macos.yml
@@ -52,7 +52,7 @@ jobs:
         run: |
           brew install cmake googletest bash rename libogg libvorbis glew \
                        openal-soft sdl2 sdl2_image sdl2_ttf freetype harfbuzz \
-                       fribidi glib gtk-doc glbinding libraqm glm zlib
+                       fribidi glib gtk-doc glbinding@2 libraqm glm zlib
           install_name_tool -change \
             '@rpath/libsharpyuv.0.dylib' \
             /usr/local/opt/webp/lib/libsharpyuv.0.dylib \

